docs/paas-plus/dev/troubleshooting/the-process-exceeded-the-timeout.md

```diff
@@ -0,0 +1,27 @@
+---
+title: The process exceeded the timeout
+description: Know what to do if you get the timeout error in Spryker Code Upgrader
+template: concept-topic-template
+---
+
+If the execution of Spryker Code Upgrader returns a Composer timeout error, make sure that your access token is still valid. It may be expired or have incorrect permissions.
```

Comment:
```suggestion
If the execution of Spryker Code Upgrader returns a Composer timeout error, make sure that your access token is still valid. It may be expired or have incorrectly set permissions.
```